compositeviews/dataset.json

```diff
@@ -0,0 +1,58 @@
+{
+  "@id": "connectome-test-search-composite-01",
+  "@type": "CompositeView",
+  "sources": [
+    {
+      "@id": "test-source-01",
+      "@type": "ProjectEventStream"
+    }
+  ],
+  "projections": [
+    {
+      "@id": "connectome-projection-composite-01",
+      "@type": "ElasticSearchProjection",
+      "mapping": {
```

Comment:
Bitte index settings noch aufnehmen (für Konfiguration mehrsprachigkeit)

Reply:
Could you indicate a link to the ES docs where this is described?

Reply:
https://www.elastic.co/blog/multilingual-search-using-language-identification-in-elasticsearch?